From fc18a73d6ed1995aabb3149df2a970af383ea1a7 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Wed, 28 Mar 2018 16:00:44 +0300
Subject: [PATCH 411/741] staging: fsl-dpaa2/eth: Add CEETM option

commit 9cec7f725c72302c1671810369806b44b3e423ee from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Add a Kconfig option for enabling CEETM support in the DPAA2
Ethernet driver.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/staging/fsl-dpaa2/Kconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index 0403f8eacd67..b29b35c4490f 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -46,6 +46,13 @@ config FSL_DPAA2_ETH_DCB
           (PFC) in the driver.
 
           If unsure, say N.
+
+config FSL_DPAA2_ETH_CEETM
+	depends on NET_SCHED
+	bool "DPAA2 Ethernet CEETM QoS"
+	default n
+	---help---
+	  Enable QoS offloading support through the CEETM hardware block.
 endif
 
 source "drivers/staging/fsl-dpaa2/mac/Kconfig"
-- 
2.17.1

